core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -24,7 +24,9 @@ class PolicyManager {
     var numberOfAdoptedPolicies = 0
     var shouldOpenPolicyPicker = false
         get() = field && canAdoptPolicy()
-    var legalismState = HashMap<String, String>()
```

Comment:
This is serialized in existing save games - Typically we have conversion code somewhere in game load or settransients for a few versions - Just mentioning in case you didn't think of this

Reply:
Good point, didn't think about that